src/kernel.cpp

```diff
@@ -36,18 +36,18 @@ static std::map<int, unsigned int> mapStakeModifierCheckpoints =
 // Get time weight
 int64_t GetWeight(int64_t nIntervalBeginning, int64_t nIntervalEnd)
 {
-    return nIntervalEnd - nIntervalBeginning - nStakeMinAge;
+    return nIntervalEnd - nIntervalBeginning - Params().StakeMinAge();
```

Comment:
I went digging to see if there was any division by GetWeight (to see if nStakeMinAge going to zero created a potential division by zero issue), but instead:

Is GetWeight() used anywhere?

Reply:
Nope. Can be removed.